### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by desojo**
Edited description and included a link to the original fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ For interactionless use, set `ALLOWLISTED_PLUGINS=example-plugin1,example-plugin
 
 | Plugin       | Description     |
 |--------------|-----------|
-| Twitter      | AutoGPT is capable of retrieving Twitter posts and other related content by accessing the Twitter platform.|
+| [desojo/autogpt-twitter](https://github.com/desojo/autogpt-twitter)      | Gives Auto-GPT the ability to interact with twitter users via the v1.1 API using Tweepy|
```
Comment (hdkiller) [inline README.md:11]: Why do you link the original fork if new code will be maintained here?

Comment (Wladastic) [reply README.md:11]: Maybe it would make sense to link the original fork in the third party list and the official version up here?

Comment (ntindle) [reply README.md:11]: I’d say don’t do this, the goal of including twitter here was to make it a central source of code for twitter plugins. Linking externally doesn’t help that

**Review by ntindle — APPROVED** [review]

**Comment by ntindle** [comment]
Approved but we should work towards getting twitter support for v2 even if the pricing is dumb

**Comment by desojo** [comment]
I can agree with that, since v1.1 is apparently on life support. If someone could make a new issue for it [that would be great](http://atlassianblog.wpengine.com/wp-content/uploads/Bill-Thatd-be-great.png)

**Comment by ntindle** [comment]
#11

**Review by ntindle — APPROVED** [review]

**Pull request merged**